src/containers/WelcomePage/components/LMASubjects.tsx

```diff
@@ -169,10 +182,18 @@ const LMASubjects = ({ ndlaId }: Props) => {
             />
             <ControlWrapperDashboard>
               <SubjectDropdown
-                subjectIds={subjectIds || []}
+                subjectIds={allSubjectIds || []}
```

Comment:
Hvor brukes subjectIds nå isåfall?

Reply:
Det er den filtrerte lista med id'er som trengs for å filtrere i spørringen, fjernet den herfra fordi vi ikke ønsker å filtrere i lista som vises i selecten!